src/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj

```diff
@@ -57,7 +57,7 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="UnicodeData.txt" />
+    <EmbeddedResource Include="$(CommonTestPath)\Data\UnicodeData.txt" />
```

Comment:
could you make this have the logical name too and change the code as needed which reading it from System.Text.Encodings.Web? I expect after this change this test will fail if you didn't change the resource reading code